Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/NoEmptyBlockConfiguration.swift

```diff
@@ -0,0 +1,26 @@
+import SwiftLintCore
+
+@AutoApply
+struct NoEmptyBlockConfiguration: SeverityBasedRuleConfiguration {
+    typealias Parent = NoEmptyBlockRule
+
+    @MakeAcceptableByConfigurationElement
+    enum CodeBlockType: String, CaseIterable {
+        case accessorBodies = "accessor_bodies"
+        case functionBodies = "function_bodies"
+        case initializerBodies = "initializer_bodies"
+        case statementBlocks = "statement_blocks"
+
+        static let all = Set(allCases)
+    }
+
+    @ConfigurationElement(key: "severity")
+    private(set) var severityConfiguration = SeverityConfiguration<Parent>(.warning)
+
+    @ConfigurationElement(key: "disabled")
+    private(set) var disabledBlockTypes: [CodeBlockType] = []
```

Comment:
Please let the key's name coincide with the variable name:
```suggestion
    @ConfigurationElement(key: "disabled_block_types")
    private(set) var disabledBlockTypes: [CodeBlockType] = []
```
Later, we want to have the keys be inferred from the variable names.